app/api/email/route.js

```diff
@@ -0,0 +1,54 @@
+import { NextResponse } from "next/server";
+import pb from '../../Domain/pocketbase';
+import nodemailer from 'nodemailer';
+
+export async function POST(request) {
+    const data = await request.json();
+    const { email, name, cart } = data;
+
+    if (email == '' || name == '' || !cart) {
+        return NextResponse.json('Missing Information', { status: 400 });
+    }
+
+    const emailPattern = /^[a-z0-9.]+@[a-z0-9]+\.[a-z]+(\.[a-z]+)?$/i;
+
+    if (!emailPattern.test(email)) {
+        return NextResponse.json('Invalid Email', { status: 400 });
+    }
+
+    let cartItemsHtml = '';
+    for (const item of cart) {
+        cartItemsHtml += `<p>${item.title}</p>`;
+    }
+
+    const html = `
+        <p>Hello ${name},</p>
+        <p>Thank you for reserving your purchase. To complete your transaction, simply visit the ACM office to make your payment and pick up your products.</p>
+        <p>Here are the items you have reserved:</p>
+        ${cartItemsHtml}
+    `;
+
```

Comment:
We need to move email api login data to .env variable

EX:
in the env file - VAR_NAME=value
in api - process.env.VAR_NAME